### DISCUSSION TITLE
engineering-practices-warnings-information-messages

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#23] [Technical Blog]
# engineering-practices-warnings-information-messages

https://wsbctechnicalblog.github.io/engineering-practices-warnings-information-messages.html

**Comment by wpschaub** [comment]
As a software engineer you are strapped inside a Dragon capsule, connected atop a mighty SpaceX Falcon Heavy Rocket. You are staring at the main monitor, observing the last 30 second countdown and you notice some mere warnings pop up. What would your reaction be if the launch director chose to push the go-for-launch button?

As mentioned by author, we should all prioritise quality from the start and treat warnings as errors, before they turn into errors.
